src/midiEditorCommands.cpp

```diff
@@ -2292,6 +2293,94 @@ void postToggleFunctionKeysAsStepInput(int command) {
 	}
 }
 
+void postMidiToggleMute(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int eventCount = countSelectedEvents(take);
+	if (eventCount == 0) {
+		return;
+	}
+	// Get selected notes.
+	vector<MidiNote> selectedNotes = getSelectedNotes(take);
+	// Get selected CCs.
+	vector<MidiControlChange> selectedCCs = getSelectedCCs(take);
+	int noteCount = selectedNotes.size();
+	int CCCount = selectedCCs.size();
+	ostringstream s;
+	if (CCCount == 0) { // If only notes are selected
+		auto chord = findChord(take, 0, {
+			true,  // start
+			true,  // end
+			true,  // channel
+			true,  // pitch
+			true,  // velocity
+			false,  // selected
+			true  // muted
+		});
+		// Select and play the current chord.
+		vector<MidiNote> notes(chord.first, chord.second);
+		previewNotes(take, notes);
+		fakeFocus = FOCUS_NOTE;
+		if (settings::reportNotes) {
+			if (noteCount == 1) {
+				if (selectedNotes[0].muted)
+				s << translate("muted") << " ";
+				else
+				s << translate("unmuted") << " ";
+				s << getMidiNoteName(take, selectedNotes[0].pitch, selectedNotes[0].channel);
+			} else {
+				// Translators: used when reporting the number of notes.
+				// {} will be replaced by the number of notes. E.g. "3 notes"
+			s << format(translate("{} notes"), noteCount);
+				int mutedCount = count_if(selectedNotes.begin(), selectedNotes.end(), [](auto note) { return note.muted; });
+				if (mutedCount > 0) {
+					// Translators: used when reporting the number of muted notes.
+					// {} will be replaced by the number of muted notes. E.g. "3 muted"
+					s << " " << format(
+					translate_plural("{} muted", "{} muted", mutedCount), mutedCount);
```

Comment:
I realise this is consistent with the messaging for muted notes when we report chords, but I worry this is confusing here. If I understand correctly, this might result in something like "7 notes 3 muted", which really means "3 notes muted 4 notes unmuted". It's unclear in this context what "7 notes" means. In the case of a chord, you're navigating to it, so "7 notes" means the total number of notes in the chord. But when you're toggling mute, "7 notes" is unqualified.